android/app/src/main/java/app/priceguard/ui/additem/setprice/SetTargetPriceFragment.kt

```diff
@@ -84,10 +82,30 @@ class SetTargetPriceFragment : Fragment() {
         tvSetPriceCurrentPrice.contentDescription =
             getString(R.string.current_price_info, tvSetPriceCurrentPrice.text)
 
-        setTargetPriceViewModel.setProductInfo(productCode, title, price)
-        etTargetPrice.setText(targetPrice.toString())
+        rsTargetPrice.setMaxPercentValue(MAX_PERCENT)
+        rsTargetPrice.setStepSize(STEP_SIZE)
 
-        updateSlideValueWithPrice(targetPrice.toFloat())
+        btnTargetPriceDecrease.setOnClickListener {
+            val sliderValue = if (rsTargetPrice.sliderValue >= STEP_SIZE) {
+                rsTargetPrice.sliderValue - STEP_SIZE
+            } else {
+                0
+            }
```

Comment:
```suggestion
            val sliderValue = (rsTargetPrice.sliderValue - STEP_SIZE).coerceIn(0, MAX_PERCENT)
```

위와 비슷한 방식으로 한줄로 줄일 수 있을 것 같습니다.